locales/de.json

```diff
@@ -0,0 +1,623 @@
+    },
+    "group-labelledby": {
+      "pass": "",
+      "fail": "Die Elemente mit dem Namen \"{{=it.data.name}}\" referenzieren mit aria-labelledby nicht alle dasselbe Element."
+    },
+    "accesskeys": {
+      "pass": "",
+      "fail": "Das Dokument enthält mehrere Elemente mit demselben accesskey-Attribut."
+    },
+    "focusable-no-name": {
+      "pass": "",
+      "fail": "Das Element befindet sich in der Tabreihenfolge und enthält keinen erreichbaren Text."
+    },
+    "landmark-is-top-level": {
+      "pass": "",
+      "fail": "Die {{=it.data.role }} landmark befindet sich innerhalb einer anderen landmark."
+    },
+    "page-has-heading-one": {
+      "pass": "",
+      "fail": "Die Seite muss eine Überschrift erster Ebene besitzen."
+    },
+    "page-no-duplicate-banner": {
+      "pass": "",
+      "fail": "Das Dokument besitzt mehr als eine banner landmark."
+    },
+    "page-no-duplicate-contentinfo": {
+      "pass": "",
+      "fail": "Das Dokument besitzt mehr als eine contentinfo landmark."
+    },
+    "page-no-duplicate-main": {
+      "pass": "",
+      "fail": "Das Dokument besitzt mehr als eine main landmark."
+    },
+    "tabindex": {
+      "pass": "",
+      "fail": "Das Element besitzt einen Tab-Index größer als 0."
+    },
+    "duplicate-img-label": {
+      "pass": "",
+      "fail": "Das Element besitzt ein <img>-Element mit Alternativtext, der vorhandenen Text wiederholt."
+    },
+    "explicit-label": {
+      "pass": "",
+      "fail": "Das Formular-Element besitzt kein explizites <label>."
+    },
+    "help-same-as-label": {
+      "pass": "",
+      "fail": "Der Hilfstext (angegeben durch ein title- oder aria-describedby-Attribut) entspricht dem Label-Text."
+    },
+    "implicit-label": {
+      "pass": "",
+      "fail": "Das Formular-Element besitzt kein implizites <label>."
+    },
+    "multiple-label": {
+      "pass": "",
+      "fail": "Das Formular-Element besitzt mehrere <label>-Elemente."
+    },
+    "title-only": {
+      "pass": "",
+      "fail": "Das Formular-Element ist lediglich durch ein title-Attribut beschriftet."
+    },
+    "has-lang": {
+      "pass": "",
+      "fail": "Das <html>-Element besitzt kein lang-Attribut."
+    },
+    "valid-lang": {
+      "pass": "",
+      "fail": "Der Wert des lang-Attributes ist nicht valide."
+    },
+    "dlitem": {
+      "pass": "",
+      "fail": "Der Definitionslisteneintrag besitzt kein <dl>-Elternelement."
+    },
+    "has-listitem": {
+      "pass": "",
+      "fail": "Das Listen-Element besitzt kein <li>-Element."
+    },
+    "listitem": {
+      "pass": "",
+      "fail": "Der Listeneintrag besitzt kein <ul>-, <ol>- oder role=\"list\" Elternelement."
+    },
+    "only-dlitems": {
+      "pass": "",
+      "fail": "Das Definitionslisten-Element enthält unerlaubte Kindelemente."
```

Comment:
`<dl>-Element` perhaps? I know the English just says `List element`

Reply:
Yes, <dl>-Element is better for consistency